clang/test/CodeGenSPIRV/spirv-builtin-lookup-intel-subgroup.cpp

```diff
@@ -0,0 +1,252 @@
+// RUN: %clang_cc1 -triple=spir64 -fdeclare-spirv-builtins -emit-llvm %s -o - | FileCheck %s
```

Comment:
The test is in CodeGenSPIRV, yet the used triple spir64, not spirv. Is that intended?

Reply:
done, changed to spirv64. I was using spir64 to get spir mangling.